### ISSUE TITLE
Childless node won't stay on selected level.

### ISSUE THREAD

**Issue by aserafim19** [sirolf2009/fxgraph#3]
Hey there,

First of all, thank you so much for your work on fxgraph! I was wondering if you can tell me how to modify the code so that a childless node can stay on any level instead of just on the most-bottom level?

For example:
In my second level here, the node on the left has 2 children whereas the node on the right has 3. 
![image](https://user-images.githubusercontent.com/49638638/60755205-a4f30900-9fec-11e9-8eaa-26f3ba506b48.png)

As soon as I remove the 2 children of the left node, the graph transforms into this:
![image](https://user-images.githubusercontent.com/49638638/60755230-0ca95400-9fed-11e9-8248-751a10fdb783.png)

I would like for that left node in the second level to remain on the second level with the right node and for the 3 children of the right node to remain on the third level.

Would this be possible? Any help and advice you can give is greatly appreciated. Thanks again!

**Comment by sirolf2009** [comment]
Yeah I had come across the same issue. I used https://github.com/abego/treelayout for the graph layout, but it looks like I'll have to create my own. I'm rather busy with work and life atm, so unfortunately it won't be soon

**Comment by Col-E** [comment]
Curious, how are you handling node removal?
